src/transformers/commands/pt_to_tf.py

```diff
@@ -171,9 +190,7 @@ def get_image_inputs(self):
 
     def run(self):
         # Fetch remote data
-        # TODO: implement a solution to pull a specific PR/commit, so we can use this CLI to validate pushes.
```

Comment:
And here you shoud raise an explicit error if the version of huggingface_hub is not enough (unless no PR will be made of course).